angelsrefining/prototypes/override/refining-override-sorting.lua

```diff
@@ -536,10 +537,10 @@ OV.patch_recipes(
           ["!!"] = (not special_vanilla) and {true, true, true, true},
           ["copper-ore"] = (not special_vanilla) and {2, 3, 4, 4},
           ["tin-ore"] = (not special_vanilla) and {2, 2, 2, 2},
-          ["quartz"] = (not special_vanilla) and {0, 1, 1, 1},
+          ["silver-ore"] = new_s and {0, 1, 1, 1},
```

Comment:
new_s is always nil, shouldn't this be
```suggestion
          ["silver-ore"] = (not special_vanilla) and {0, 1, 1, 1},
```